### PR TITLE
TOC and index for Docker items

### PR DESCRIPTION
**Pull request by spboyer**
# Docker TOC item
## Summary
- Added Docker to /core/docker/index.md
- Added index.md and moved latest Docker article in TOC 
## Suggested Reviewers

@BillWagner

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.runtime.compilerservices.callconvthiscall?branch=pr-en-us-990)

File: docs/toc.md [hunk 1]
```diff
@@ -275,12 +275,13 @@
 ### [Developing ASP.NET Core applications](core/tutorials/aspnet-core.md)
 ### [How to Manage Package Dependency Versions for .NET Core 1.0](core/tutorials/managing-package-dependency-versions.md)
 ### [Using MSBuild to build .NET Core projects](core/tutorials/target-dotnetcore-with-msbuild.md)
-### [Building Docker Images for .NET Core Applications](core/docker/building-net-docker-images.md)
 ## [App Types](core/app-types.md)
 ## [Packages, Metapackages and Frameworks](core/packages.md)
 ## [Deploying](core/deploying/index.md)
 ### [🔧 Deploying Applications](core/deploying/applications.md)
 ### [Creating a NuGet Package with Cross Platform Tools](core/deploying/creating-nuget-packages.md)
+## Docker (core/docker/index.md)
```
Comment (BillWagner) [inline docs/toc.md]: The [ ] are missing around [Docker]. After that, it's good to go.

File: docs/core/docker/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+manager: wpickett
+ms.date: 09/01/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: cae32148-aef4-4e64-a7f0-88072bad4400
+---
+
+Docker and .NET Core
```
Comment (BillWagner) [inline docs/core/docker/index.md]: nit: we're standardizing on the 

# Docker and .NET Core

syntax for headers instead of the underline.

**Comment by BillWagner** [comment]
Looks great. Fix the two markup related comments I left, and then let's merge.

**Comment by spboyer** [comment]
@BillWagner look good to go

**Comment by BillWagner** [comment]
Awesome. I'll look once the build finishes.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.security.cryptography.rsaopenssl?branch=pr-en-us-990)

**Comment by BillWagner** [comment]
All good. Let's :shipit:

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.servicemodel.channels.ioutputsessionchannel?branch=pr-en-us-990)

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/windows.ui.xaml.automation.elementnotenabledexception?branch=pr-en-us-990)

**Pull request merged**